htdocs/don/list.php

```diff
@@ -193,12 +199,16 @@
     $searchpicto=$form->showFilterAndCheckAddButtons(0);
     print $searchpicto;
     print '</td>';
-	print "</tr>\n";
-
-	print '<tr class="liste_titre">';
+    print "</tr>\n";
+    
+    print '<tr class="liste_titre">';
 	print_liste_field_titre("Ref", $_SERVER["PHP_SELF"], "d.rowid", "", $param, "", $sortfield, $sortorder);
+    if (! empty($conf->global->DONATION_USE_THIRDPARTIES)) {
+	print_liste_field_titre("ThirdParty", $_SERVER["PHP_SELF"], "d.fk_soc", "", $param, "", $sortfield, $sortorder);
+    } else {
 	print_liste_field_titre("Company", $_SERVER["PHP_SELF"], "d.societe", "", $param, "", $sortfield, $sortorder);
 	print_liste_field_titre("Name", $_SERVER["PHP_SELF"], "d.lastname", "", $param, "", $sortfield, $sortorder);
```

Comment:
The name of the donator/contact can be kept event when option DONATION_USE_THIRDPARTIES is on ?
The only change should be to use the key instead of a free text, shouldn't it ?

Reply:
@eldy i keep donator name just replace company / thirdparty. Is it ok ?